OCPBUGS-30910: Clarify IP address node limitations (#93659)
openshift/openshift-docs: modules/nw-egress-ips-about.adoc

IP capacity = public cloud default capacity - sum(current IP assignments)
----

> While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, if a public cloud provider limits IP address capacity to 10 IP addresses per node, the total number of assignable IP addresses is only 80. To increase this to 150 IP addresses per node, you would need to allocate 7 additional nodes.

Reviewer: `the total number of assignable IP addresses is only 80`, where does this number `80` come from?

Author: @huiran0826, oops, I've updated this. Thanks!

> While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, if a public cloud provider limits IP address capacity to 10 IP addresses per node and there are 8 nodes in the cluster, the total number of assignable IP addresses is only 80. To increase this to 150 IP addresses per node, you would need to allocate 7 additional nodes.

Reviewer 2: Can reword to something similar as below for simplicity. Need to fix the future tense though.

```suggestion
While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, if a public cloud provider limits IP address capacity to 10 IP addresses per node, for a cluster with 8 nodes, the total number of assignable IP addresses is only 80. To increase this to 150 IP addresses per node, you must allocate 7 additional nodes.
```
